maintainers/maintainer-list.nix

```diff
@@ -3135,7 +3251,7 @@
   infinisil = {
     email = "contact@infinisil.com";
     github = "infinisil";
-    githubId = 20525370;
+    #githubId = 20525370;
```

Comment:
Why is it commented out?

Reply:
This was a test to see whether ofborg would catch it (seems to have worked, but it's not very nice (also impure), so I'll change this)